### PR TITLE
refactor(examples): clean up no-param-reassign violations

### PR DESCRIPTION
**Pull request by sangxxh**
Issue #990 

Refactor code to remove `// eslint-disable-next-line no-param-reassign` in `examples`

### CODE REVIEW

File: examples/data-objects/table-document/src/cellrange.ts [hunk 1]
```diff
@@ -21,12 +21,12 @@ export function colNameToIndex(colName: string) {
 export function colIndexToName(colIndex: number) {
     let name = "";
 
+    let i = colIndex;
     do {
         const mod = colIndex % 26;
```
Comment (curtisman) [inline examples/data-objects/table-document/src/cellrange.ts:26]: should this `colIndex` be `i`?

Comment (curtisman) [inline examples/data-objects/table-document/src/cellrange.ts:26]: should this `colIndex` be `i`

**Review by curtisman — CHANGES_REQUESTED** [review]

File: examples/data-objects/table-document/src/cellrange.ts (hunk 1 above)
Comment (hantatsang) [reply examples/data-objects/table-document/src/cellrange.ts]: Thanks

**Review by curtisman — APPROVED** [review]
:shipit:

**Comment by curtisman** [comment]
@hantatsang Thank you for the PR!

**Pull request merged**